### ISSUE TITLE
fitColorCurve: column not found in table

### ISSUE THREAD

**Issue by Zeklandia** [jpierel14/snsed#6]
While running the tutorial code (without print calls), fitColorCurve crashed:
```
Getting best fit for: U-B,r-J,r-H,r-K
No model provided, running series of models.
Best fit model is "snana-2004fe", with a Chi-squared of 300.254609
Running BIC for color:
     U-B
Traceback (most recent call last):
  File "…/snsedextend_tutorial.py", line 18, in <module>
    curveDict = snsedextend.fitColorCurve(colorTable)
  File "…/anaconda3/envs/astro3/lib/python3.7/site-packages/snsedextend/cc_extrap.py", line 660, in fitColorCurve
    tempTable=Table([tempTable['time'],tempTable[color],tempTable[color[0]+color[-1]+'_err'],tempTable['SN']],names=('time','mag','magerr','SN'))
  File "…/anaconda3/envs/astro3/lib/python3.7/site-packages/astropy/table/table.py", line 1346, in __getitem__
    return self.columns[item]
  File "…/anaconda3/envs/astro3/lib/python3.7/site-packages/astropy/table/table.py", line 190, in __getitem__
    return OrderedDict.__getitem__(self, item)
KeyError: 'SN'
```

**Comment by Zeklandia** [comment]
When I run it as part of what I'm working on, it says it is attempting to U-B also, but the color table only has r-z. The crash is identical.

**Comment by jpierel14** [comment]
Should be resolved in commit f05047a82909c184107657e5274a43c67521e173
